BUILD.gn

```
@@ -126,6 +126,8 @@ vvl_sources = [
   "layers/core_checks/cc_spirv.cpp",
   "layers/core_checks/cc_state_tracker.cpp",
   "layers/core_checks/cc_state_tracker.h",
+  "layers/core_checks/cc_submit_time_validation.cpp",
```

```suggestion
  "layers/core_checks/cc_submit_time.cpp",
```

I feel it gets slippery to start calling any file in the Validation Layers as `validation`

---

cc_submit

---

Updated